airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/manifest.yaml

```diff
@@ -1515,6 +1491,120 @@ definitions:
+      - comments
+      - companyPageClicks
+      - conversionValueInLocalCurrency
+      - costInLocalCurrency
+      - costInUsd
+      - dateRange
+      - documentCompletions
+      - documentFirstQuartileCompletions
+      - documentMidpointCompletions
+      - documentThirdQuartileCompletions
+      - downloadClicks
+      - externalWebsiteConversions
+      - externalWebsitePostClickConversions
+      - externalWebsitePostViewConversions
+      - follows
+      - fullScreenPlays
+      - impressions
+      - jobApplications
+      - jobApplyClicks
+      - landingPageClicks
+      - leadGenerationMailContactInfoShares
+      - leadGenerationMailInterestedClicks
+      - likes
+      - oneClickLeadFormOpens
+      - oneClickLeads
+      - opens
+      - otherEngagements
+      - pivotValues
+      - postClickJobApplications
+      - postClickJobApplyClicks
+      - postClickRegistrations
+      - postViewJobApplications
+      - postViewJobApplyClicks
+      - postViewRegistrations
+      - reactions
+      - registrations
+      - sends
+      - shares
+      - talentLeads
+      - textUrlClicks
+      - totalEngagements
+      - validWorkEmailLeads
+      - videoCompletions
+      - videoFirstQuartileCompletions
+      - videoMidpointCompletions
+      - videoStarts
+      - videoThirdQuartileCompletions
+      - videoViews
+      - viralCardClicks
+      - viralCardImpressions
+      - viralClicks
+      - viralCommentLikes
+      - viralComments
+      - viralCompanyPageClicks
+      - viralDocumentCompletions
+      - viralDocumentFirstQuartileCompletions
+      - viralDocumentMidpointCompletions
+      - viralDocumentThirdQuartileCompletions
+      - viralDownloadClicks
+      - viralExternalWebsiteConversions
+      - viralExternalWebsitePostClickConversions
+      - viralExternalWebsitePostViewConversions
+      - viralFollows
+      - viralFullScreenPlays
+      - viralImpressions
+      - viralJobApplications
+      - viralJobApplyClicks
+      - viralLandingPageClicks
+      - viralLikes
+      - viralOneClickLeadFormOpens
+      - viralOneClickLeads
+      - viralOtherEngagements
+      - viralPostClickJobApplications
+      - viralPostClickJobApplyClicks
+      - viralPostClickRegistrations
+      - viralPostViewJobApplications
+      - viralPostViewJobApplyClicks
+      - viralPostViewRegistrations
+      - viralReactions
+      - viralRegistrations
+      - viralShares
+      - viralTotalEngagements
+      - viralVideoCompletions
+      - viralVideoFirstQuartileCompletions
+      - viralVideoMidpointCompletions
+      - viralVideoStarts
+      - viralVideoThirdQuartileCompletions
+      - viralVideoViews
+    # dateRange and pivotValue fields are used to merge records back together so they must be in every API request
+    always_include_properties:
+      - dateRange
+      - pivotValues
+    property_chunking:
+      type: PropertyChunking
+      property_limit_type: property_count
+      property_limit: 18
```

Comment:
Is the property_limit 18 because of dateRange and pivotValue which are not included in the count to 20?

Reply:
yes that's correct. the actual max is 20, but we must include those two values in every request in order to merge records together correctly

Reply:
It feels like it would have been a better UX to have the property chunking consider the values that are always present so that the limit aligns with the API restrictions. For property_count, it is probably fine but for number of bytes, it becomes a bit more annoying to calculate. Overall, it is probably a nit but still wanted to raise my concern here